Shaders/Materials/BlinnPhong/BlinnPhong.frag.glsl

```diff
@@ -12,12 +11,17 @@ layout (location = 2) in vec3 in_normal;
 layout (location = 3) in vec3 in_tangent;
 layout (location = 4) in vec3 in_viewVector;
 layout (location = 5) in vec3 in_lightVector;
+layout (location = 6) in vec3 in_vertexcolor;
+
+#include "BlinnPhongMaterial.glsl"
```

Comment:
same here, why moving the include after the layouts declaration?

Reply:
Because a bad BSDF interface design (blend between varying and uniform parameters in some BSDF related functions). To be corrected in V2.